go/cache/theine/singleflight_test.go

```diff
@@ -98,7 +98,7 @@ func TestDoDupSuppress(t *testing.T) {
 	c <- "bar"
 	wg2.Wait()
 	got := atomic.LoadInt32(&calls)
-	assert.True(t, got > 0 && got < n, "number of calls not between 0 and %d", n)
+	assert.True(t, got > 0 && got < n)
```

Comment:
`assert.Greater` and `assert.Lesser` might produce more useful error messages

Reply:
ah, these are mostly from using [test writer](https://github.com/arvind-murty/test-rewriter) tool you mentioned. I'll go back and fix the tests

Reply:
yeah that tool isn't perfect but it's definitely a start over doing this all by hand! just needs some touching up after